doc/man3/RAND_add.pod

```diff
@@ -54,6 +57,11 @@ should consider using L<RAND_load_file(3)> instead.
 
 RAND_seed() is equivalent to RAND_add() with B<randomness> set to B<num>.
 
+RAND_keep_random_devices_open() use used to control the resource use
+of the random device seed sources. Some seed sources maintain open file
+descriptors which allow them to operate in chroot(2) environment without
```

Comment:
"chroot(2) environment" is misleading, because "environment" gives an impression of something where you can start multiple processes. While the only thing that is effective in the context of proposed modification is operation within **single** process boundary that itself called chroot(2). Even more specifically, chroot(2) has to be called *after* rand is initialized.